documentation/assemblies/assembly-connecting-applications.adoc

```diff
@@ -19,8 +19,54 @@ To connect to the messaging service from outside the {KubePlatform} cluster, TLS
 
 The messaging protocols supported depends on the type of address space used. For more information about address space types, see  link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#con-address-space-messaging[Address space].
 
-//TODO: Add information about retrieving the CA certificate and using
-//that in the client examples.
+[id='client-getcer-{context}']
+== Retrieving the Self Signed CA Certificate
+
+If you opted for the `selfsigned` certificate provider type in your addressspace endpoint configuration, the generated CA
+that signed the adddressspace server certificate is required to when connecting messaging client application. The
+certificate can be retrieved from the addressspace using the following procedure.
+
+WARNING: The use of selfsigned certificates is not recommended for use in production environments.
+
+.Procedure
+
+ifeval::["{cmdcli}" == "oc"]
+. Log in as a messaging tenant:
++
+[subs="attributes",options="nowrap"]
+----
+{cmdcli} login -u developer
+----
+endif::[]
+
+. Retrieve the CA certificate from the addressspace required
++
```

Comment:
Are these "++" required here?

Reply:
The first plus belongs to https://www.gnu.org/software/diffutils/manual/diffutils.html#Detailed-Unified, the second is an [Ascii doc list continuation](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#lists).

Reply:
Looks good. Thanks for the information.